src/com/jalasoft/search/controller/SearchCriteria.java

```diff
@@ -0,0 +1,321 @@
+/*
+ *  Copyright (c) 2018 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information").  You shall not disclose such Confidential Information and shall use
+ *   it only in accordance with the terms of the license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.search.controller;
+
+/**
+ * SearchCriteria : This class is a container class when will be stored the search criteria
+ *
+ * @author rafael alfaro
+ * @version 1.0
+ */
+
+public class SearchCriteria {
+
+    private String driver;
+    private String searchPath;
+    private String fileName;
+    private String advanceSearch;
+    private String contains;
+    private boolean inTitle;
+    private boolean insideFile;
+    private String ownerFile;
+    private boolean fileCreated;
+    private boolean fileModified;
+    private boolean fileAccessed;
+    //    private Date initDate;
+//    private Date finalDate;
```

Comment:
Remove these lines if these are not required in the code

Reply:
The lines 31 and 32 were removed.